scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+#
+# This workflow:
+#  - Does conversion of output seg files to a format compatible with IGV.  This will be removed soon, since the GATK tools
+#     have been updated to produce files that can be easily read by IGV.
+#  - Tags possible germline events that appear in the tumor results.  This is done by comparing breakpoints and
+#     reciprocal overlaps.  This algorithm takes into account any hypersegmentation.
+#  - Tags tumor events that appear in the given blacklists (centromere and gistic blacklist)
+#  - Removes any events that have been tagged in the previous two steps.  Any resulting gaps are covered, so long as
+#     the segment mean would remain constant and the distance is less than max_merge_distance (default 1Mb as of
+#     this writing)
+#
+#  Outputs:
+#  - seg file that is has germline events removed (and gaps merged).  Same columns as a called seg file.
+#      This should be considered the final output.  IGV compatible.  The results have been evaluated.
+#  - seg file that can be used as input to ABSOLUTE.  This uses an identical format as AllelicCapSeg.  This is NOT compatible with IGV.
+#      Please note that the balanced-segment calling in this file have not been evaluated heavily.
+#  - skew file for the skew parameter in ABSOLUTE
+#  - skew as a float for the skew parameter in ABSOLUTE
+#  - Intermediate files for browsing various steps of the workflow.
+#
+# Important notes:
+#  - The probe and het counts become inaccurate in the results.  These need to be corrected, but that is not trivial.
+#  - This workflow has not been tested with hg38
+#  - Evaluation of germline tagging and blacklists on hg38 is still pending.
+#  - Evaluation of the conversion to ACS format for ABSOLUTE is still pending.
+#  TODO: Post the auxiliary files in a public bucket.  Permission has already been granted.
+workflow CombineTracksWorkflow {
+	File tumor_called_seg
+	File tumor_modeled_seg
+	File af_param
+	File matched_normal_called_seg
+    File ref_fasta
+    File ref_fasta_dict
+    File ref_fasta_fai
+    File centromere_tracks_seg
+    File gistic_blacklist_tracks_seg
+    File? gatk4_jar_override
+    Array[String] columns_of_interest
+    Int? germline_tagging_padding
+    String group_id
+    String gatk_docker
+    Int? max_merge_distance
+
+    call CombineTracks {
+        input:
+            tumor_called_seg = tumor_called_seg,
+            matched_normal_called_seg = matched_normal_called_seg,
+            centromere_tracks_seg = centromere_tracks_seg,
+            columns_of_interest = columns_of_interest,
+            ref_fasta = ref_fasta,
+            ref_fasta_dict = ref_fasta_dict,
+            ref_fasta_fai = ref_fasta_fai,
+            germline_tagging_padding = germline_tagging_padding,
+            gistic_blacklist_tracks_seg = gistic_blacklist_tracks_seg,
+            gatk4_jar_override = gatk4_jar_override,
+            gatk_docker = gatk_docker
+    }
+
+    call IGVConvert as IGVConvertNormal {
```

Comment:
Again, if we no longer need these tasks, perhaps we should just remove them now, since it seems like a significant fraction of the code in this PR deals with the IGV conversion.  Will leave it up to you.

Reply:
Done where trivially possible.